### ISSUE TITLE
Loading shader fails at runtime

### ISSUE THREAD

**Issue by thewtex** [thewtex/craco-vtk#1]
Running *craco-vtk/test-app* with `npm start`, it builds, but during runtime with this traceback:

![image](https://user-images.githubusercontent.com/25432/56046165-a45e1580-5d10-11e9-8ea1-9cdfcd3e0db5.png)

and this error in the console:

```
index.js:1446 Error compiling shader '/static/media/vtkPolyDataVS.a68a7aff.glsl': ERROR: 0:1: '/' : syntax error

index.js:1446 0: /static/media/vtkPolyDataVS.a68a7aff.glsl
```

the content of */static/media/vtkPolyDataVS.a68a7aff.glsl* is

```
module.exports = "module.exports = \"//VTK::System::Dec\\n\\n/*=========================================================================\\n\\n  Program:   Visualization Toolkit\\n  Module:    vtkPolyDataVS.glsl\\n\\n  Copyright (c) Ken Martin, Will Schroeder, Bill Lorensen\\n  All rights reserved.\\n  See Copyright.txt or http://www.kitware.com/Copyright.htm for details.\\n\\n     This software is distributed WITHOUT ANY WARRANTY; without even\\n     the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR\\n     PURPOSE.  See the above copyright notice for more information.\\n\\n=========================================================================*/\\n\\nattribute vec4 vertexMC;\\n\\n// frag position in VC\\n//VTK::PositionVC::Dec\\n\\n// optional normal declaration\\n//VTK::Normal::Dec\\n\\n// extra lighting parameters\\n//VTK::Light::Dec\\n\\n// Texture coordinates\\n//VTK::TCoord::Dec\\n\\n// material property values\\n//VTK::Color::Dec\\n\\n// clipping plane vars\\n//VTK::Clip::Dec\\n\\n// camera and actor matrix values\\n//VTK::Camera::Dec\\n\\n// Apple Bug\\n//VTK::PrimID::Dec\\n\\n// picking support\\n//VTK::Picking::Dec\\n\\nvoid main()\\n{\\n  //VTK::Color::Impl\\n\\n  //VTK::Normal::Impl\\n\\n  //VTK::TCoord::Impl\\n\\n  //VTK::Clip::Impl\\n\\n  //VTK::PrimID::Impl\\n\\n  //VTK::PositionVC::Impl\\n\\n  //VTK::Light::Impl\\n\\n  //VTK::Picking::Impl\\n}\\n\""
```

CC: @floryst @jourdain

**Comment by thewtex** [comment]
:xref: https://github.com/Kitware/vtk-js/issues/1042 which is the same issue.

**Comment by thewtex** [comment]
Addressed by a386069bf8ab6151afef506642f7e509b10030ae
